api/subscriptions/subscriptions_test.go

```diff
@@ -59,7 +59,7 @@ func TestDeprecatedSubscriptions(t *testing.T) {
 
 	_, resp, err := websocket.DefaultDialer.Dial(u.String(), nil)
 	assert.Error(t, err)
-	assert.Equal(t, http.StatusGone, resp.StatusCode)
+	assert.Equal(t, http.StatusNotFound, resp.StatusCode)
```

Comment:
Can we leave the gone status like before? It looks more precise when telling something that was there before but now it's been deprecated

Reply:
Yep, good call, fixed 👍